### PR TITLE
use custom entry point for webpack

### PR DESCRIPTION
**Pull request by mgax**
So we can better control what happens in the build without having to learn arcane webpack apis and plugins.

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    function(_, name) { return options[name] })
+}
+
+webpack(WEBPACK_OPTIONS, function(err, stats) {
+  if(err) { return console.error(err) }
+  console.log(stats.toString("normal"))
+
+  var t = '' + new Date().getTime()
+  for(let name of ['/index.html', '/doc.html', '/terms.html']) {
+    var data = render(fs.readFileSync(SRC_DIR + name, 'utf8'), {t: t})
```
Comment (gabriel-v) [inline build.js]: Wouldn't `{t}` work here instead of `{t: t}`?

Comment (gabriel-v) [inline build.js]: As `t` will be an everlasting argument given to all of our home-made templates, i think we should use something less cryptic, like `timestamp`.

It should be obvious to an unknowing third party what that template parameter is set to.

Comment (mgax) [inline build.js]: Renaming to `timestamp`. I'm afraid the shorthand `{t}` syntax might not work, this code is run straight up by node, it's not compiled with babel.

**Comment by mgax** [comment]
> This may lack some of the shiny reports

Yeah, there's several output options, but I was unable to reproduce the output from the command-line version, it was either too verbose or too terse. So much magic in the JS ecosystem :/

**Pull request merged**